entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity_data_request.proto

```diff
@@ -100,3 +100,18 @@ message MergeAndUpsertEntityRequest {
 message MergeAndUpsertEntityResponse {
   Entity entity = 1;
 }
+
+message DeleteEntitiesRequest {
+  string entity_type = 1;
+  oneof delete_request {
+    StringList entity_ids = 2;
```

Comment:
Filter is an attribute filter. we don't push api id as attribute in api entity

Reply:
Otherwise i would have to add special handling for this logic in transform(AttributeFilter) function

Reply:
ugh, right. I hate entity service

Reply:
what if we add the delete API in entity query service, instead of entity data service. We do support ID as an attribute there. And, the delete API can then just work out of `entity_type` and attribute filter

Reply:
https://github.com/hypertrace/entity-service/pull/182#discussion_r908418106